docs/content/manual/pre-release/ha/partial-engine-deployment.md

```diff
@@ -22,7 +22,7 @@ https://github.com/longhorn/longhorn/issues/2081
 1. Detach `vol-1`
 1. Verify that Longhorn cannot attach `vol-1` to `node-1` since there is no engine image on `node-1`
 1. Check that all functions (attach to other nodes, detach, snapshot, backup, expand, restore, creating DR volume, ... ) are working ok for `vol-1`
-1. Verify that `vol-2` can be attached to any nodes except for `node-1`
+1. Verify that `vol-2` cannot be attached to any nodes because one of its replica is sitting on the `node-1` which doesn't have the engine image
```

Comment:
I feel this is not right and like an issue instead. 

I would assume the replica on node-1 will go to error, and the other two still running. So, the volume should be able to attach to schedulable nodes w/ degraded status.

Reply:
Yes you are right, replica on tainted become in failed status, status of remained 2 replicas on other nodes were in stopped status, I logged https://github.com/longhorn/longhorn/issues/4386 for tracking

Reply:
Why the other running replicas will become stopped? cc @shuo-wu @c3y1huang @PhanLe1010